server/middleware/must-be-authenticated.js

```diff
@@ -3,5 +3,7 @@ module.exports = function mustBeAuthenticated(req, res, next) {
   if (req.isAuthenticated()) {
     return next();
   }
+  // TODO send 401 instead
+  // Sending redirect is old behavior, and signin may not even be available based on auth setup
   res.redirect(req.config.get('baseUrl') + '/signin');
```

Comment:
+1 to send 401 instead. I was also coming across with this when working on PR #588

Reply:
😅 yeah I think it'll be a separate PR. This tripped me up too during testing.

There's actually a lot of API stuff I'd like to fix. I'm kind of thinking about making a list and making it a v5 thing?

Reply:
v5 is fine, we can also document the API and every endpoint. I know it's a good amount of work :)